aspnetcore/blazor/webassembly-lazy-load-assemblies.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Lazy load assemblies in ASP.NET Core Blazor WebAssembly
+author: guardrex
+description: Discover how to lazy load assemblies in ASP.NET Core Blazor WebAssembly apps.
+monikerRange: '>= aspnetcore-5.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/16/2020
+no-loc: [Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/webassembly-lazy-load-assemblies
+---
+# Lazy load assemblies in ASP.NET Core Blazor WebAssembly
+
+By [Safia Abdalla](https://safia.rocks) and [Luke Latham](https://github.com/guardrex)
+
+Blazor WebAssembly app startup performance can be improved by loading assets at runtime when the assets are required, which is called *lazy loading*. For example, assemblies that are only used to render a single component can be set up to load only if the user navigates to that component. After loading, the assemblies are cached client-side and don't require reloading while the app is running.
```

Comment:
```suggestion
Blazor WebAssembly app startup performance can be improved by loading assets at runtime when the assets are required, which is called *lazy loading*. For example, assemblies that are only used to render a single component can be set up to load only if the user navigates to that component. After loading, the assemblies are cached client-side and are available for all future navigations.
```

Reply:
I guess reloading has very specific meaning in browser \ webassembly, we might want to avoid conflating the two.